compile-css.js

```diff
@@ -20,70 +20,17 @@ const isProd = process.env.ELEVENTY_ENV === 'prod';
 const fs = require('fs');
 const path = require('path');
 const log = require('fancy-log');
-
-const sassEngine = (function() {
-  try {
-    // node-sass is faster, but regularly fails to install correctly (native bindings)
-    return require('node-sass');
-  } catch (e) {
-    // fallback to the official transpiled version
-    return require('sass');
-  }
-})();
-
-/**
- * @param {string} input filename to read for input
- * @param {string} output filename to use for output (but does not write)
- * @return {{css: !Buffer, map: !Buffer}}
- */
-function compileCSS(input, output) {
```

Comment:
The change here is that I've moved this helper into "src/build/compile-css.js" so we can share with both the top-level CSS compilation and CSS compilation for components.